server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/delivery/SieveExecutor.java

```diff
@@ -55,15 +54,9 @@ public static Builder builder() {
         private MailetContext mailetContext;
         private UsersRepository usersRepos;
         private SievePoster sievePoster;
-        private String folder;
         private ResourceLocator resourceLocator;
         private Log log;
 
-        public Builder folder(String folder) {
```

Comment:
Giving a default folder in case Sieve script execution fails in not a Sieve responsibility.

We should let the Delivery logic decide for this.